package.json

```diff
@@ -29,12 +29,13 @@
   "homepage": "https://github.com/stampit-org/stamp-specification#readme",
   "devDependencies": {
     "babel-cli": "^6.6.5",
-    "babel-eslint": "^5.0.0",
+    "babel-eslint": "^6.1.2",
     "babel-polyfill": "^6.7.2",
     "babel-preset-es2015": "^6.6.0",
     "babel-register": "^6.7.2",
-    "check-compose": "^2.0.0",
-    "eslint": "~2.2.0"
+    "check-compose": "^3.1.2",
+    "eslint": "^2.13.1",
```

Comment:
@boneskull Well if you write client side code only it's fine, but isomorphic apps are more then required these days.